VHbbAnalysis/Heppy/test/vhbb.py

```diff
@@ -208,16 +232,15 @@ class TestFilter(logging.Filter):
     def filter(self, record):
         print record
 
-
-
 # and the following runs the process directly 
 if __name__ == '__main__':
     from PhysicsTools.HeppyCore.framework.looper import Looper 
-    looper = Looper( 'Loop', config, nPrint = 20, nEvents = 300)
+    looper = Looper( 'Loop', config, nPrint = 1, nEvents = 100)
+
     import time
     import cProfile
     p = cProfile.Profile(time.clock)
     p.runcall(looper.loop)
     p.print_stats()
-#    looper.loop()
+    looper.loop()
```

Comment:
Right, sorry. I had initially removed the profiler and re-added it back, missing to remove the last call to looper.loop(). Will remove.